src/pages/library/DataBrowserPreview.js

```diff
@@ -118,9 +126,11 @@ const DataBrowserPreview = ({ id }) => {
 
   useOnMount(() => {
     const loadData = async () => {
-      const metadata = await Ajax(signal).DataRepo.getPreviewMetadata(id)
-      setTables(metadata.tables)
-      selectTable({ value: metadata.tables[0].name })
+      if (snapshot && snapshot.access === snapshotAccessTypes.GRANTED) {
```

Comment:
can this be written using `?.`?
```suggestion
      if (snapshot?.access === snapshotAccessTypes.GRANTED) {
```

Reply:
yes, i'll make that change 👍